hooks/post_gen_project.sh

```diff
@@ -9,7 +9,12 @@
 set -euo pipefail
 
 echo "Fetching latest 'pants' script"
-curl -L -o ./pants https://pantsbuild.github.io/setup/pants
+curl --proto "=https" \
+     --tlsv1.0 \
```

Comment:
This command appears to work with 1.3

Reply:
Yeah, I'm trying to make this a bit more accessible (also trying to make similar contributions to Pants itself). `--tlsv1.0` sets a floor on the TLS version that will be used, but that also depends on what your libraries have been compiled to support. `curl` on my laptop has apparently been compiled with 1.3 support, but in CI, it hasn't.

If your `curl` has been compiled with 1.3 support, then 1.3 will be used with this invocation. If you've only got 1.2, however, it'll use 1.2, since that is greater than 1.0.

I'm still tweaking this, though.